haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -13,16 +13,19 @@ have to click \"refresh\" or anything first)
 - BeShare supports any number of simultaneous uploads and downloads \
 in parallel, but will serialize simultaneous file requests to any \
 given host for efficiency."
+HOMEPAGE="https://github.com/HaikuArchives/BeShare"
 COPYRIGHT="1999-2012 Jeremy Friesner"
 LICENSE="
 	Public Domain
 	libsanta
 	"
-HOMEPAGE="https://github.com/HaikuArchives/BeShare"
-SOURCE_URI="$(HOMEPAGE)/archive/b721fcc.tar.gz"
+SOURCE_URI="${HOMEPAGE}/archive/b721fcc.tar.gz"
```

Comment:
`2345e48664fddfff1d9a968d1da2a185dbbdb50c` is still categorized beshare  2.31 isn it?